### ISSUE TITLE
Meeting List configuration causes critical error after Bread update

### ISSUE THREAD

**Issue by jjustinj68** [bmlt-enabled/bread#219]
Since the 2.8.3 update out meeting list will not generate.  It causes a Wordpress critical error to be generated instead.  This happens when using the PDF link on our site (https://dev.coastalcarolinaarea.org/?current-meeting-list=3) as well as when in Bread and trying to use the 'Preview' or 'Generate Meeting List' from within Bread Configuration Manager or Customizer.

If I revert back to the default Bread configuration the issue is not present.

We are going to be printing schedules within a week or so.  I have created a new configuration based on default and saved it and have started modifying to our needs.  In the Meetings 'tab' of Customizer, Bread does not appear to be working properly with respect to changes to settings of 'Group Meetings By' nor 'Display subgrouping' .  Wonderng if those settings in our configuration is where error is being generated from?

I had an exported copy of our configuration file and I tried to import that to see if it helped.  It did not.  I am including a copy of that configuration file with this report. 

This started happening after Bread updated to 2.8.3 and continues in 2.8.4. 

Debug Information
Bread Version:
Wordpress Version: 6.7.1
Protocol: https://
PHP Version: 8.1.30
Server Version: LiteSpeed
Temporary Directory: /tmp/

(Also just noticed Bread version info is not being populated in the Debug Information I just copied from Bread)

Please advise asap as again, we are going to be printing soon.

Kind Regards
Justin J
CCANA  website coordinator

[coastal-carolina-area-of-narcotics-anonymous-ccana-setting-2-011-14-2024.json](https://github.com/user-attachments/files/18137138/coastal-carolina-area-of-narcotics-anonymous-ccana-setting-2-011-14-2024.json)

**Comment by otrok7** [comment]
Hi Justin,  Thanks for this complete bug report.  There seems to be a build problem, and the Arial fonts are broken.  I will investigate, it probably won't be more than a few days.  In the meantime, if you switch your font to "DejaVu Sans Condensed", you should be fine.  Just an interim solution, of course.  I will get the Arial fonts up again....

**Comment by jjustinj68** [comment]
Thank you for the quick response!
Changing the font to DejaVu Sans Condensed did fix the crashing, thank you.

I'll look for the update in a few days regarding the headers issue .  If there is anything I can do to be of assistance, please do not hesitate to ask.  

Thank you for your service!!!

Justin J

**Comment by jjustinj68** [comment]
Arial is working again, thank you!

The issue I also reported about in the Meetings 'tab' of Customizer, Bread does not appear to be working properly with respect to changes to settings of 'Group Meetings By' nor 'Display subgrouping' is still not working,  at least not with the setting we use. 

More specifically,  when 'Group Meetings By' is set to 'Weekday+City' and 'Display subgrouping'  is set to 'Combine main and subgroup into a single header' it only has headers for City and there are no subgroup headers at all. 

 If 'Display subgrouping'  is set to 'Display each subgroup with it's own header' it does display a main header for the Day and then each City is subgroup with header in the meeting list section.

Thank you again for your service and as always if I can do anything to be of assistance, please let me know.

**Comment by otrok7** [comment]
OK, fix on its way.

**Comment by jjustinj68** [comment]
Just noticed another discrepancy.  For some reason the (Cont) header isn't working for Wednesday only.  Attaching screenshot for reference
![BreadHeaderBug](https://github.com/user-attachments/assets/fcf15aa2-7a46-4d9f-b347-f05e01658b96)

**Comment by otrok7** [comment]
It's because there is a subheading there.  The logic is, if there's a heading, any heading, then the (cont) heading is not required.  I might rethink that.  Have you now moved away from combined headings?  Or did the last release not fix the problem?

**Comment by jjustinj68** [comment]
I just changed to subheadings for now so our schedule on the website looked OK until fix for combined headings was available.   The body at PR preferred the combined headings so if that can be an option we will change back.  

I can see your point on the combined heading not being there if a subheading is.   I just noticed it because it just happens that all out other columns have combined heading at the top so it looks different.  Certainly not a show stopper though...call that one an enhancement request lol.
